docs/release-notes/deepspeed-trainer.rst

```diff
@@ -0,0 +1,13 @@
+:orphan:
+
+**New Features**
+
+-  DeepSpeed Trainer: Add Trainer API to DeepSpeedTrial. Users can now iterate on DeepSpeed models
+   off-cluster. An example of DeepSpeed Trainer API has been added in the `examples/deepspeed/dcgan`
```

Comment:
```suggestion
   off-cluster. An example of DeepSpeed Trainer API has been added in the ``examples/deepspeed/dcgan``
```
Sorry, I'm not too familiar with RST, but was this meant as a `code literal`? If not, please ignore.

Same question for the ones in the Breaking Changes section.